README.md

```diff
@@ -42,7 +42,7 @@ directory will be created with the generated project name. For example:
 cd ~/devel
 cookiecutter gh:frequenz-floss/frequenz-repo-config-python \
     --directory=cookiecutter \
-    --checkout v0.5.2
+    --checkout v0.6.2
```

Comment:
Can we actually have a CI checker script that makes sure that this number is always pointing to the latest release?

Reply:
Even better, I just created this issue: https://github.com/frequenz-floss/frequenz-repo-config-python/issues/138

Reply:
I guess it should run on tags only, and check that the `pyproject.toml` template is using the version that is being built.

Reply:
If not a PR, a issue would be also welcome.